### PR TITLE
Added dropdown

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/801

#### Describe the changes you have made in this pr -
- Added dropdown in place of Notifications

### Working -

CircuitVerse-Online-Digital-Logic-Circuit-Simulator

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@MrBartusek Done ! Updated the working gif

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -31,10 +31,22 @@
         <a class="nav-link text-dark" href="#">About</a>
       </li>
       <% if user_signed_in? %>
+       <div class="navbar-nav nav-item dropdown">
         <li class="nav-item px-2">
-          <a class="nav-link text-dark" href="<%= notifications_path(current_user) %>">
-            Notifications
-          </a>
+          <a class="nav-link text-dark dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (JoshHeng) [inline app/views/layouts/_header.html.erb]: As @MrBartusek has said, use `style="cursor: pointer;"` instead of `href="#"`

**Comment by Ananthavijay** [comment]
Done @JoshHeng @MrBartusek

**Comment by aayush-05** [comment]
This looks good to be merged @tachyons

**Pull request merged**